disputer/disputer.js

```diff
@@ -61,13 +61,27 @@ class Disputer {
       });
 
       // Create the liquidation transaction
+      const dispute = this.empContract.methods.dispute(disputeableLiquidation.id, disputeableLiquidation.sponsor);
 
-      // TODO: compute the amount of collateral required to dispute and ensure the bot has enough to perform it.
+      // Simple version of inventory management: simulate the transaction and assume that if it fails, the caller didn't have enough collateral.
+      try {
+        await dispute.call({ from: this.account });
+      } catch (error) {
+        Logger.warn({
```

Comment:
Do we want to print the `error` to the console? We probably would not send it to Slack.

Reply:
Maybe it should be `info` instead? I spoke with @chrismaree this morning about how we should think about log levels. He told me that as of current writing, we use 3 log levels: `error`, `info`, and `debug`. `error` and `info` will go to Slack and console. `debug` will only go to the console.

I think this log, in particular *should* go to slack because it implies that the dispute bot cannot do its job due to lack of funds, which is something we/another dispute bot runner would probably want to rectify ASAP or risk allowing an invalid liquidation to go through undisputed. The one downside I see is that this error will spam slack each time it runs since it's not sporadic and won't resolve itself.

Thoughts?

Reply:
I was referring to the actual exception thrown, the `error` object. We should print that somewhere, *at least* to the console.

Reply:
Oooooh, I totally misunderstood. SGTM. Will add.